### PR TITLE
zh: update scaling document

### PR DESCRIPTION
**Pull request by Yisaer**
### Which TiDB Operator version(s) do your changes apply to? (Required)
update scaling document

- [x] master (the latest development version, including v1.1 changes for now)

### CODE REVIEW

File: zh/scale-a-tidb-cluster.md [hunk 1]
```diff
@@ -57,15 +62,19 @@ TiDB 水平扩缩容操作指的是通过增加或减少节点的数量，来达
...
-3. 查看升级进度：
+### 垂直扩缩容操作 (yaml)
 
-    {{< copyable "shell-regular" >}}
+1. 通过 kubectl 修改集群所对应的 `TidbCluster` 对象的 `spec.pd.resources`、`spec.tikv.resources`、`spec.tidb.resources` 至期望值。
 
-    ```shell
-    watch kubectl -n <namespace> get pod -o wide
-    ```
+### 查看升级进度：
```
Comment (ran-huang) [inline zh/scale-a-tidb-cluster.md]:
```suggestion
### 查看升级进度
```

File: zh/scale-a-tidb-cluster.md [hunk 2]
```diff
@@ -10,9 +10,10 @@ category: how-to
 
 ## 水平扩缩容
 
-TiDB 水平扩缩容操作指的是通过增加或减少节点的数量，来达到集群扩缩容的目的。扩缩容 TiDB 集群时，会按照填入的 replicas 值，对 PD、TiKV、TiDB 进行顺序扩缩容操作。扩容操作按照节点编号由小到大增加节点，缩容操作按照节点编号由大到小删除节点。
+TiDB 水平扩缩容操作指的是通过增加或减少节点的数量，来达到集群扩缩容的目的。扩缩容 TiDB 集群时，会按照填入的 replicas 值，对 PD、TiKV、TiDB 进行顺序扩缩容操作。扩容操作按照节点编号由小到大增加节点，缩容操作按照节点编号由大到小删除节点。目前 TiDB 集群有通过 helm 部署与使用 Yaml 文件直接部署两种方式，你可以根据 TiDB 集群的部署方式选择对应的方式进行伸缩。
```
Comment (DanielZhangQD) [inline zh/scale-a-tidb-cluster.md]:
```suggestion
TiDB 水平扩缩容操作指的是通过增加或减少节点的数量，来达到集群扩缩容的目的。扩缩容 TiDB 集群时，会按照填入的 replicas 值，对 PD、TiKV、TiDB 进行顺序扩缩容操作。扩容操作按照节点编号由小到大增加节点，缩容操作按照节点编号由大到小删除节点。目前 TiDB 集群有通过 Helm 与使用 TidbCluster Custom Resource(CR) 两种管理方式，你可以根据 TiDB 集群的管理方式选择对应的方式进行伸缩。
```

File: zh/scale-a-tidb-cluster.md [hunk 3]
```diff
@@ -10,9 +10,10 @@ category: how-to
...
+
+### 水平扩缩容操作 (helm)
```
Comment (DanielZhangQD) [inline zh/scale-a-tidb-cluster.md]:
```suggestion
### 水平扩缩容操作 (Helm)
```

File: zh/scale-a-tidb-cluster.md [hunk 4]
```diff
@@ -24,15 +25,19 @@ TiDB 水平扩缩容操作指的是通过增加或减少节点的数量，来达
     helm upgrade <release-name> pingcap/tidb-cluster -f values.yaml --version=<chart-version>
     ```
 
-3. 查看集群水平扩缩容状态：
+### 水平扩缩容操作 (yaml)
```
Comment (DanielZhangQD) [inline zh/scale-a-tidb-cluster.md]:
```suggestion
### 水平扩缩容操作 (CR)
```

File: zh/scale-a-tidb-cluster.md [hunk 5]
```diff
@@ -24,15 +25,19 @@ TiDB 水平扩缩容操作指的是通过增加或减少节点的数量，来达
...
 
-    {{< copyable "shell-regular" >}}
+1. 使用 kubectl 修改集群所对应的 `TidbCluster` 对象中的 `spec.pd.replicas`、`spec.tidb.replicas`、`spec.tikv.replicas` 指期望值。
```
Comment (DanielZhangQD) [inline zh/scale-a-tidb-cluster.md]:
```suggestion
1. 使用 kubectl 修改集群所对应的 `TidbCluster` 对象中的 `spec.pd.replicas`、`spec.tidb.replicas`、`spec.tikv.replicas` 至期望值。
```

Comment (DanielZhangQD) [reply zh/scale-a-tidb-cluster.md]: Give an example command here?

File: zh/scale-a-tidb-cluster.md [hunk 6]
```diff
@@ -43,9 +48,9 @@ TiDB 水平扩缩容操作指的是通过增加或减少节点的数量，来达
 
 ## 垂直扩缩容
```
Comment (DanielZhangQD) [inline zh/scale-a-tidb-cluster.md]: The comments above are also applicable to this section.

**Comment by Yisaer** [comment]
@ran-huang @DanielZhangQD  PTAL

File: zh/scale-a-tidb-cluster.md [hunk 7]
```diff
@@ -10,9 +10,10 @@ category: how-to
 
 ## 水平扩缩容
 
-TiDB 水平扩缩容操作指的是通过增加或减少节点的数量，来达到集群扩缩容的目的。扩缩容 TiDB 集群时，会按照填入的 replicas 值，对 PD、TiKV、TiDB 进行顺序扩缩容操作。扩容操作按照节点编号由小到大增加节点，缩容操作按照节点编号由大到小删除节点。
+TiDB 水平扩缩容操作指的是通过增加或减少节点的数量，来达到集群扩缩容的目的。扩缩容 TiDB 集群时，会按照填入的 replicas 值，对 PD、TiKV、TiDB 进行顺序扩缩容操作。扩容操作按照节点编号由小到大增加节点，缩容操作按照节点编号由大到小删除节点。目前 TiDB 集群有通过 Helm 与使用 TidbCluster Custom Resource(CR) 两种管理方式，你可以根据 TiDB 集群的管理方式选择对应的方式进行伸缩。
```
Comment (ran-huang) [inline zh/scale-a-tidb-cluster.md]:
```suggestion
TiDB 水平扩缩容操作指的是通过增加或减少节点的数量，来达到集群扩缩容的目的。扩缩容 TiDB 集群时，会按照填入的 replicas 值，对 PD、TiKV、TiDB 进行顺序扩缩容操作。扩容操作按照节点编号由小到大增加节点，缩容操作按照节点编号由大到小删除节点。目前 TiDB 集群有通过 Helm 与使用 TidbCluster Custom Resource (CR) 两种管理方式，你可以根据 TiDB 集群的管理方式选择对应的方式进行伸缩。
```

File: zh/scale-a-tidb-cluster.md [hunk 8]
```diff
@@ -24,15 +25,19 @@ TiDB 水平扩缩容操作指的是通过增加或减少节点的数量，来达
     helm upgrade <release-name> pingcap/tidb-cluster -f values.yaml --version=<chart-version>
     ```
 
-3. 查看集群水平扩缩容状态：
+### 水平扩缩容操作 (CR)
 
-    {{< copyable "shell-regular" >}}
+1. 使用 kubectl 修改集群所对应的 `TidbCluster` 对象中的 `spec.pd.replicas`、`spec.tidb.replicas`、`spec.tikv.replicas` 至期望值。
```
Comment (ran-huang) [inline zh/scale-a-tidb-cluster.md]:
```suggestion
使用 kubectl 修改集群所对应的 `TidbCluster` 对象中的 `spec.pd.replicas`、`spec.tidb.replicas`、`spec.tikv.replicas` 至期望值。
```

File: zh/scale-a-tidb-cluster.md [hunk 9]
```diff
@@ -43,9 +48,9 @@ TiDB 水平扩缩容操作指的是通过增加或减少节点的数量，来达
 
 ## 垂直扩缩容
 
-垂直扩缩容操作指的是通过增加或减少节点的资源限制，来达到集群扩缩容的目的。垂直扩缩容本质上是节点滚动升级的过程。
+垂直扩缩容操作指的是通过增加或减少节点的资源限制，来达到集群扩缩容的目的。垂直扩缩容本质上是节点滚动升级的过程。目前 TiDB 集群有通过 helm 部署与使用 Yaml 文件直接部署两种方式，你可以根据 TiDB 集群的部署方式选择对应的方式进行伸缩。
```
Comment (ran-huang) [inline zh/scale-a-tidb-cluster.md]:
```suggestion
垂直扩缩容操作指的是通过增加或减少节点的资源限制，来达到集群扩缩容的目的。垂直扩缩容本质上是节点滚动升级的过程。目前 TiDB 集群有通过 Helm 部署与使用 yaml 文件直接部署两种方式，你可以根据 TiDB 集群的部署方式选择对应的方式进行伸缩。
```

File: zh/scale-a-tidb-cluster.md [hunk 10]
```diff
@@ -57,15 +62,19 @@ TiDB 水平扩缩容操作指的是通过增加或减少节点的数量，来达
     helm upgrade <release-name> pingcap/tidb-cluster -f values.yaml --version=<chart-version>
     ```
 
-3. 查看升级进度：
+### 垂直扩缩容操作 (CR)
 
-    {{< copyable "shell-regular" >}}
+1. 通过 kubectl 修改集群所对应的 `TidbCluster` 对象的 `spec.pd.resources`、`spec.tikv.resources`、`spec.tidb.resources` 至期望值。
```
Comment (ran-huang) [inline zh/scale-a-tidb-cluster.md]:
```suggestion
通过 kubectl 修改集群所对应的 `TidbCluster` 对象的 `spec.pd.resources`、`spec.tikv.resources`、`spec.tidb.resources` 至期望值。
```

File: zh/scale-a-tidb-cluster.md [hunk 11]
```diff
@@ -24,15 +25,19 @@ TiDB 水平扩缩容操作指的是通过增加或减少节点的数量，来达
...
+### 查看集群水平扩缩容状态
+
+{{< copyable "shell-regular" >}}
+
+```shell
+watch kubectl -n <namespace> get pod -o wide
+```
 
-    当所有组件的 Pod 数量都达到了预设值，并且都进入  `Running` 状态后，水平扩缩容完成。
+当所有组件的 Pod 数量都达到了预设值，并且都进入  `Running` 状态后，水平扩缩容完成。
```
Comment (DanielZhangQD) [inline zh/scale-a-tidb-cluster.md]:
```suggestion
当所有组件的 Pod 数量都达到了预设值，并且都进入 `Running` 状态后，水平扩缩容完成。
```

File: zh/scale-a-tidb-cluster.md [hunk 12]
```diff
@@ -43,9 +48,9 @@ TiDB 水平扩缩容操作指的是通过增加或减少节点的数量，来达
 
 ## 垂直扩缩容
 
-垂直扩缩容操作指的是通过增加或减少节点的资源限制，来达到集群扩缩容的目的。垂直扩缩容本质上是节点滚动升级的过程。
+垂直扩缩容操作指的是通过增加或减少节点的资源限制，来达到集群扩缩容的目的。垂直扩缩容本质上是节点滚动升级的过程。目前 TiDB 集群有通过 Helm 部署与使用 yaml 文件直接部署两种方式，你可以根据 TiDB 集群的部署方式选择对应的方式进行伸缩。
```
Comment (DanielZhangQD) [inline zh/scale-a-tidb-cluster.md]:
```suggestion
垂直扩缩容操作指的是通过增加或减少节点的资源限制，来达到集群扩缩容的目的。垂直扩缩容本质上是节点滚动升级的过程。目前 TiDB 集群有通过 Helm 与使用 TidbCluster Custom Resource (CR) 两种管理方式，你可以根据 TiDB 集群的管理方式选择对应的方式进行伸缩。
```

**Comment by Yisaer** [comment]
@ran-huang PTAL

**Pull request merged**